### DISCUSSION TITLE
Error running my first coin tutorial

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#52] [Questions] [answered]
Asked by `vpawar46` in discord

> Hey, I'm new to the Aptos blockchain and while running pnpm run your_coin, facing issues like " Module not found " &  ELIFECYCLE  Command failed with exit code 1.

**Comment by 0x-j** [comment, marked as answer]
1. We are moving away from coin and we recommend all devs to use the [fungible asset standard](https://aptos.dev/standards/fungible-asset) going forward. Here's a [tutorial to create a fungible asset](https://aptos.dev/tutorials/your-first-fungible-asset).
2. When you run examples in aptos-ts-sdk/example directory, you need to first build the sdk by running `pnpm build` at the root directory. Then run the example like `pnpm run your_fungible_asset`. I tried the coin example and it works after I build the sdk first.

**Reply by 0x-j** [reply]
I updated the [tutorial to include the instructions on building sdk first](https://aptos.dev/tutorials/your-first-coin#step-3-run-the-example) before running example.

**Reply by vinaypawar01** [reply]
Yeah it works, we first need to build. Thanks!
